src/logging.h

```diff
@@ -91,7 +91,7 @@
   if (DWG_LOGLEVEL >= DWG_LOGLEVEL_##level && wstr) { \
     BITCODE_TU ws = wstr;                             \
     uint16_t _c;                                      \
-    while ((_c = *ws++)) {                            \
+    while (ws && (_c = *ws++)) {                      \
```

Comment:
Not needed. We already checked wstr in line 91

Reply:
Yor are right. I'll remove this.